deploy/v1.4.0/polygon.ts

```diff
@@ -0,0 +1,109 @@
+import { dim } from 'chalk';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { deployAndLog } from '../../src/deployAndLog';
+import { setManager } from '../../src/setManager';
+import { transferOwnership } from '../../src/transferOwnership';
+
+export default async function deployToPolygonMainnet(hre: HardhatRuntimeEnvironment){
+    if (process.env.DEPLOY === 'v1.4.0.polygon') {
+        dim(`Deploying: PrizeTierHistory Polygon Mainnet`)
+        dim(`Version: 1.4.0`)
+    } else { return }
+    const { deployer, executiveTeam, ptOperations, defenderRelayer } = await hre.getNamedAccounts();
+    const drawCalculator = await hre.ethers.getContract('DrawCalculator');
+
+    const prizeTierHistory = await hre.ethers.getContract('PrizeTierHistory');
+    const lastPrizeTier = await prizeTierHistory.getPrizeTier(await(prizeTierHistory.getNewestDrawId()));
+
+     /* PrizeTierHistory ------------------ */
+    // @dev Required by PrizeDistributionFactory
+    /* ----------------------------------- */
+    await deployAndLog('PrizeTierHistory', {
+        from: deployer,
+        args: [deployer],
+        skipIfAlreadyDeployed: false,
+    });
+    const prizeTierHistoryNew = await hre.ethers.getContract('PrizeTierHistory');
+    await prizeTierHistoryNew.push(lastPrizeTier)
+    
+    /* PrizeDistributionFactory ---------- */
+    // @dev Immutable reference to new PrizeTierHistory
+    // @dev Use existing references for the remaining contracts
+    // @dev Required by ReceiverTimelockTrigger
+    /* ----------------------------------- */
+    const drawBuffer = await hre.ethers.getContract('DrawBuffer');
+    const prizeDistributionBuffer = await hre.ethers.getContract('PrizeDistributionBuffer');
+    const ticket = await hre.ethers.getContract('Ticket');
+    const minPickCost = 1000000
```

Comment:
We could use the constant `PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST` from `src/constants.ts`.